app/install/aws-linux.md

```diff
@@ -54,7 +54,7 @@ baseurl=https://kong.bintray.com/kong-rpm/amazonlinux/amazonlinux
     $ sudo yum install -y kong
     ```
 
-2. **Prepare your database**
+2. **Prepare your database or declarative configuration file**
 
     [Configure][configuration] Kong so it can connect to your database. Kong supports both [PostgreSQL {{site.data.kong_latest.dependencies.postgres}}](http://www.postgresql.org/) and [Cassandra {{site.data.kong_latest.dependencies.cassandra}}](http://cassandra.apache.org/) as its datastore.
```

Comment:
If the database is optional, we shouldn't begin by telling people "Configure Kong so it can connect to your database" and then after all is said, add that "you can skip the steps above" afterwards.

Reply:
(Added a suggestion above)